src/test/CMakeLists.txt

```diff
@@ -268,6 +268,7 @@ set(
     # Added plugin targets so that we can test member methods without going through dlsym
     hyriseMvccDeletePlugin
     hyriseUccDiscoveryPlugin
+    # hyriseBenchmarkLib
```

Comment:
?

Reply:
The outcommented cmake-stuff and the outcommented includes are an artifact from playing around to fix the CI stage, gonna remove.

Reply:
Just uncommented everything again (required for testing the benchmark hooks). Let's see if it also passes CI. 🤞